src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  return function adder(number) {
+    if (number === undefined) {
```

Comment:
or you can just check 
```suggestion
    if (!number) {
```

Reply:
![image](https://user-images.githubusercontent.com/60401137/109476375-870acf80-7a7f-11eb-84b0-32cd80eac692.png)
это плохая проверка поскольку число может быть нулем